### PR TITLE
2019 stats

### PR DESCRIPTION
**Pull request by mhl10**
This is staged [here](https://beta.arxiv.org/help/stats/2019_by_area/).

### CODE REVIEW

File: help/stats/2019_by_area/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+<span id="astro-ph_yearly"></span>
+![astro-ph submissions by year](astro-ph_yearly.png)
+
+<span id="physics_yearly"></span>
+![physics submissions by year](physics_yearly.png)
+
+<span id="hep_yearly"></span>
+![HEP submissions by year](hep_yearly.png)
+
+<span id="quant-ph_gr-q_yearly"></span>
```
Comment (jimentwood) [inline help/stats/2019_by_area/index.md]:
```suggestion
<span id="quant-ph_gr-qc_yearly"></span>
```

File: help/stats/2019_by_area/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+*   [eess](#eess_yearly) (8819): 0.5%
+*   [q-fin](#q-fin_yearly) (7137): 0.4%
+*   [econ](#econ_yearly) (1184): 0.1%
+
+Graphs of yearly submission rates by subject areas
+--------------------------------------------------
+
+![all physics submissions by year](phys_yearly.png)
+
+<span id="cs_yearly"></span>
```
Comment (jimentwood) [inline help/stats/2019_by_area/index.md]: The anchors are rendered inline with the graphics which means and anchor link takes users to below the graphic they are looking for. This appears to be a problem with the 2018 template. Perhaps an extra line break between the span and the figure would work. In the future we may want to give text titles to the figures for improved accessibility.

Comment (mhl10) [reply help/stats/2019_by_area/index.md]: Thanks, I'll work on a fix.

Comment (mhl10) [reply help/stats/2019_by_area/index.md]: @jimentwood your suggestion worked like a charm!

**Pull request merged**